### PR TITLE
chore: Comply to Vercel Open Source sponsorship [skip release]

### LINKED ISSUE #1035: Comply to Vercel Open Source sponsorship
**Summary of proposed feature**

- A Vercel banner in the footer of each site page.
- A Vercel banner in the source repository's README.md file.
- All links back to Vercel should use a UTM tag with the format: ?utm_source=[team-name]&utm_campaign=oss

See the full list of criteria:
https://vercel.com/knowledge/can-vercel-sponsor-my-open-source-project

**Purpose of proposed feature**

We could make sure that everyone in our core team has access to the Vercel dashboard.

**Additional context**
You can find the [banner here](https://github.com/nextauthjs/next-auth/blob/canary/www/static/img/powered-by-vercel.svg).

### PR DESCRIPTION
**Pull request by suraj10k**
Closes #1035

### CODE REVIEW

File: ss [hunk 1]
```diff
@@ -0,0 +1,248 @@
+[33mcommit 7a1d7120965a01ba23e62277989f4032aa9c9916[m[33m ([m[1;36mHEAD -> [m[1;32mcanary[m[33m, [m[1;33mtag: v3.2.0-canary.14[m[33m, [m[1;31morigin/canary[m[33m, [m[1;31morigin/HEAD[m[33m)[m
```
Comment (balazsorban44) [inline ss:1]: This file should be deleted.

**Review by balazsorban44 — CHANGES_REQUESTED** [review]

**Comment by balazsorban44** [comment]
Also, look at this to how to add the banner to the site footer:

https://github.com/nextauthjs/next-auth/issues/1035#issuecomment-757906694

File: README.md [hunk 2]
```diff
@@ -140,11 +140,19 @@ export default function Component() {
 <a href="https://github.com/nextauthjs/next-auth/graphs/contributors">
   <img width="500px" src="https://contrib.rocks/image?repo=nextauthjs/next-auth" />
 </a>
+<div>
+<a href="https://vercel.com?utm_source=next-auth&utm_campaign=oss">
+<img width="170px" src="https://raw.githubusercontent.com/nextauthjs/next-auth/7a1d7120965a01ba23e62277989f4032aa9c9916/www/static/img/powered-by-vercel.svg" alt="Banner" />
```
Comment (balazsorban44) [inline README.md:145]: I would use `Powered by Vercel` as an alt text, to be more descriptive

File: ss (hunk 1 above)
Comment (suraj10k) [reply ss:1]: deleted

File: README.md [hunk 3]
```diff
@@ -140,11 +140,19 @@ export default function Component() {
...
+</a>
+</div>
+<div>
+<p allign="left">Thanks to Vercel sponsoring this project by allowing it to be deployed for free for the entire NextAuth.js Team</p>
```
Comment (balazsorban44) [inline README.md:149]: there is a typo here, it should be `align`

**Comment by balazsorban44** [comment]
Could you also remove this line:

https://github.com/nextauthjs/next-auth/blob/e7e8e0f3932eee767641ec07fc27ef16f71413eb/www/src/pages/index.js#L186

**Comment by suraj10k** [comment]
> Could you also remove this line:
> 
> https://github.com/nextauthjs/next-auth/blob/e7e8e0f3932eee767641ec07fc27ef16f71413eb/www/src/pages/index.js#L186

done , ill now work on adding the banner to the site footer

File: README.md [hunk 4]
```diff
@@ -140,11 +140,19 @@ export default function Component() {
 <a href="https://github.com/nextauthjs/next-auth/graphs/contributors">
   <img width="500px" src="https://contrib.rocks/image?repo=nextauthjs/next-auth" />
 </a>
+<div>
+<a href="https://vercel.com?utm_source=next-auth&utm_campaign=oss">
+<img width="170px" src="https://raw.githubusercontent.com/nextauthjs/next-auth/7a1d7120965a01ba23e62277989f4032aa9c9916/www/static/img/powered-by-vercel.svg" alt="Powered By Vercel" />
```
Comment (balazsorban44) [inline README.md:145]: Could you use this as a url `https://raw.githubusercontent.com/nextauthjs/next-auth/canary/www/static/img/powered-by-vercel.svg` to the banner?

**Comment by balazsorban44** [comment]
> > Could you also remove this line:
> > https://github.com/nextauthjs/next-auth/blob/e7e8e0f3932eee767641ec07fc27ef16f71413eb/www/src/pages/index.js#L186
> 
> done , ill now work on adding the banner to the site footer

Awesome, tell me when it is done, then we are good to go!

**Comment by suraj10k** [comment]
when i try to start the localhost with "npm run dev" i'm getting : "npm run dev
npm ERR! missing script: dev" . could you pls help me fix this . im new to working with react

**Comment by balazsorban44** [comment]
So to start the documentation site, you should be inside the `www` folder, run `npm i`, and run `npm start`. You will then be able to see the documentation at `http://localhost:3000`

To add the banner as a link to the footer under acknowledgments, you can see the Docosaurus docs here:
https://v2.docusaurus.io/docs/api/themes/configuration/#footer-links


I created the snippet you should add as:


```js
{html: `
  <a target="_blank" rel="noopener noreferrer" href="https://vercel.com?utm_source=next-auth&utm_campaign=oss">
    <img
      alt="Powered by Vercel"
      style="margin-top: 8px"
      height="32"
      src="https://raw.githubusercontent.com/nextauthjs/next-auth/canary/www/static/img/powered-by-vercel.svg"
    />
  </a>`
}
```

The relevant part of the file you would like to add this to is after this: https://github.com/nextauthjs/next-auth/blob/0c405295350ce211ec9bda3d63ca76fbdb3655b4/www/docusaurus.config.js#L96-L99

**Comment by suraj10k** [comment]
Thanks a lot I'll try it

**Comment by balazsorban44** [comment]
@suraj10k, thanks for your contribution!

**Comment by suraj10k** [comment]
Can I get listed as a contributor ?

**Comment by balazsorban44** [comment]
You can see yourself at this url:

https://github.com/nextauthjs/next-auth/graphs/contributors

We currently don't have anything similar to https://allcontributors.org set up.

**Comment by suraj10k** [comment]
got it ,thanks a lot, this was my first open source contribution and even thoguh it was a simple one i learnt a lot.

**Comment by balazsorban44** [comment]
Awesome! Good to hear! 🙂 Good luck on your journey! 😉

**Pull request merged**